velox/substrait/SubstraitUtils.cpp

```diff
@@ -130,6 +130,24 @@ std::string SubstraitParser::makeNodeName(int node_id, int col_idx) {
   return fmt::format("n{}_{}", node_id, col_idx);
 }
 
+int SubstraitParser::getIdxFromNodeName(const std::string& nodeName) {
```

Comment:
It would be nice to add unit tests for these methods.

Reply:
Added unit tests for these methods.